dev/reloadable-server/src/main/scala/play/core/server/LagomReloadableDevServerStart.scala

```diff
@@ -83,6 +84,14 @@ object LagomReloadableDevServerStart {
           }
         }
 
+        val before = System.currentTimeMillis()
+        val address = InetAddress.getLocalHost
+        val after = System.currentTimeMillis()
```

Comment:
Should we fallback to `System.nanoTime()` for such small measures?

Reply:
I think it is fine to use System.currentTimeMillis, the threshold of 100ms seems arbitrary as well so if we are worried about precision I'd rather change the threshold

Reply:
Yeah, I think we could maybe bump the threshold up to 500ms or 1000. I would hate for this to trigger false positives. Usually people affected by this are seeing it take several seconds.